collectors/persistentvolumeclaim.go

```diff
@@ -87,9 +98,25 @@ func (collector *persistentVolumeClaimCollector) Collect(ch chan<- prometheus.Me
 	}
 }
 
+// getPersistentVolumeClaimClass returns StorageClassName. If no storage class was
+// requested, it returns "".
+func getPersistentVolumeClaimClass(claim *v1.PersistentVolumeClaim) string {
+	// Use beta annotation first
+	if class, found := claim.Annotations[v1.BetaStorageClassAnnotation]; found {
+		return class
+	}
+
+	if claim.Spec.StorageClassName != nil {
+		return *claim.Spec.StorageClassName
+	}
+
+	return ""
```

Comment:
Updated. 
I also found "<none>" string is already used in other collectors, like [pod](https://github.com/kubernetes/kube-state-metrics/blob/master/collectors/pod.go). If it's common practice to use this non-empty special string to indicate absence of label value, maybe we need to document it somewhere, for developers and also for end-users.